File: Car Class/car_class.js
class Car{
    constructor(year, make, model) {
        this.year = year;
        this.make = make;
        this.model = model;
        this.running = false;
        this.speed = 0;
    }

    turnOn() {
        this.running = true;
    }

    turnOff() {
        this.running = false;
    }

    changeSpeed(accel) {
        this.speed += accel;
    }
}

let car1 = new Car(2021, "Dodge", "Charger-Hellcat");
let car2 = new Car(2021, "Tesla", "Model X");
let car3 = new Car(2018, "Ford", "Explorer");

let x = car1.turnOn();
let y = car1.changeSpeed(100);
let i = car1.changeSpeed(50);
let carOff = car1.turnOff;

File: Car Class/car_class.py
# Task 1
class car():
    def __init__(self, year, make, model):
        self.year = year
        self.make = make
        self.model = model
        self.running = False
        self.speed = 0
    def turOn(self):
        self.running = True
    def turnOff(self):
        self.running = False
    def changeSpeed(self, accel):
        self.speed += accel

# Task 2        
mustang1 = car (2018, "Ford", "Mustang")
ferrari1 = car (1962, "Ferrari","250 GTO")
buggati1 = car (2016, "Buggati", "Chiron")    

# Task 3, Communicate

